sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 5.7.0-beta.2 (Unreleased)
+## 5.7.0-beta.2 (2022-02-08)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- `createEventDataAdapter` is added which specifies how to convert an `EventData` to a simple message with `body` and `contentType` fields. This adapter can be used with the Avro encoder in `@azure/schema-registry-avro` starting from version 1.0.0-beta.6 to create `EventData` messages with Avro-encoded bodies.
 
 ### Other Changes
 
+- Added a sample for buffered producer. [PR #18674](https://github.com/Azure/azure-sdk-for-js/pull/18674)
```

Comment:
does this have to be mentioned? I feel like samples are not part of the package artifact so it feels weird to mention this in the changelog.

Reply:
obviously not!